schemas/pulse-job.yml

```diff
@@ -204,28 +241,180 @@ properties:
   runMachine:
     $ref: "#/definitions/machine"
 
-  artifacts:
-    type: "array"
+  textLogSummary:
```

Comment:
In the interest of moving this forward I suggest what we leave a comment in the description advertizing `textLogSummary` as an experimental property.

I can't imagine we would actually want to support this format long-term. I'm not even sure it's sane to send this much data via. pulse... AMQP performance goes down if we send big messages.